### PR TITLE
New blog post on Malawi Response

### PR DESCRIPTION
**Pull request by Sophie-GelinasGagne**

### CODE REVIEW

File: 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  
+The data gathered through these mapping efforts was used in conducting a needs assessment in Phalombe/Nkulambe, near the border of Malawi and Mozambique, where huge landslides and flash floods heavily damaged entire villages and critical infrastructure. The digitalisation of buildings allowed [MSF](https://www.facebook.com/msfinternational/?__cft__[0]=AZVbT0u2lgB6jfznJ3n1qH9KEptgu_I8GZrPsfuooKh_mRjH9aHCL_bPSSiBFxn_Ka7s11p3YgwyHhk9e9H7PfSxR-3iFgdQfPX8CwU2VJXN-qTIyXx4j9GlWQc9Q9B1_D2FTIYuas7BozT_9JIAWPyuR7UYqExea5TmNMx0RFuWWHcqKB73s7fFkdNj35uG5qY&__tn__=kK-R) to combine the OpenStreetMap (OSM) data with floods and landslide data in maps estimating the impacted zones/buildings (see map cut-out below).
```
Comment (danbjoseph) [inline 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md]: the link here is just taking me to the MSF organization page on facebook. is that how it's supposed to be working? if so, i think we can use a shorter link?

File: 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog_20230601_malawi_landslides_potential_building_damage.png">
+<p class="caption"> Malawi 2023 © MSF
+</p>
+  
+In addition, the team prioritized the verification and collection of field-specific data that couldn't be extracted solely from satellite imagery. To accomplish this, they hired a motorbike driver familiar with the area and adept at reading maps. This driver transported the team to various locations, allowing them to cross-reference village names obtained from a database with information provided by local residents. Subsequently, GIS specialist Alexis Huart incorporated the verified field data, encompassing village names, residential areas, and road segments, into the OSM database.
+  
+<figure>
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog_20230601_malawi_cyclone_freddy_impact.jpeg">
+<p class="caption">The Nkulambe health centre two weeks after the Cyclone hit. Malawi 2023 © MSF
+</p>
```
Comment (danbjoseph) [inline 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md:110]: these need a closing tag, i.e. `</figure>`

File: 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  
+For more severe cases, MSF helped facilitate referrals to larger hospitals, initially through helicopter transport and later via ambulance. Additionally, due to the lack of a maternity ward in the existing functional clinics, MSF set up maternity services in our clinic. Water and sanitation activities were also undertaken to repair damaged water pipes and boreholes, distribute chlorine, set up hand-wash stations and latrines.
+
+<figure>
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog_20230601_malawi_cyclone_freddy_MSF_WaterRepairs.jpeg">
+<p class="caption">Using all means of transport for water repairs. Malawi 2023 © MSF
+</p>
```
Comment (danbjoseph) [inline 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md:122]: same as above, needs a `</figure>`

File: 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+To date, the epidemic is still ongoing although the number of reported cases has declined significantly across the country. A total of 58,500 cases have been confirmed since the beginning of the epidemic, including 1,750 deaths.
+  
+## Cyclone Freddy causing floods and landslides
+  
+In Malawi, the seasonal occurrence of tropical cyclones in February-March has become increasingly apparent, potentially linked to the presence of climate change. The devastating aftermath of Tropical Storm Ana in 2022 prompted [disaster response mapping](https://reliefweb.int/map/malawi/malawi-phalombe-district-cyclone-ana-impact-map-31-january-2022). This year, Tropical Cyclone Freddy made a second landfall in Malawi and Mozambique in March 2023, triggering widespread flooding and devastating landslides.
+  
+<figure>
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog_20230601_malawi_cholera_response.jpeg">
+<p class="caption">The 2nd landfall of Cyclone Freddy caused extensive floods in Phalombe District. Malawi 2023 © MSF
+</p>
```
Comment (danbjoseph) [inline 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md:90]: needs a `</figure>`

File: 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+excerpt: "Malawi was experiencing the worst cholera epidemic in 20 years, so MSF sent an emergency team and activated Missing Maps. When a cyclone hit in March, OSM data was also invaluable for assessment."
+published: true
+tags: [MSF, Medecins Sans Frontieres, Health, Cholera, Malawi, OpenStreetMap, Cyclone]
+permalink: /blog/:year/:month/:day/:title/
+lang: en
+---
+
+---
+  
+# Remote Mapping in Malawi: Supporting Field Operations for the Cholera Outbreak and Cyclone Freddy Response
```
Comment (danbjoseph) [inline 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md:16]: the `title` from the metadata will display inside the image header on the blog post page. including it again here will repeat it. suggest removing if not intentional.

File: 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "Remote Mapping in Malawi: Supporting Field Operations"
+postID: 2023-06-02-Malawi-MSF-Cholera-and-Cyclone-Freddy-Response
```
Comment (danbjoseph) [inline 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md]: including the date in the `postID` will mean the URL looks like this:
https://www.missingmaps.org/blog/2023/06/02/2023-06-02-Malawi-MSF-Cholera-and-Cyclone-Freddy-Response/ (with the date in there twice)
suggest removing

**Review by danbjoseph — CHANGES_REQUESTED** [review]

File: 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md (hunk 5 above)
Comment (Sophie-GelinasGagne) [reply 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md:16]: intentional

File: 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md (hunk 1 above)
Comment (Sophie-GelinasGagne) [reply 2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md]: The link opens MSF Canada's Fb page for me, let's go with that for now - I'll double check with Jana on Monday and make changes if needed.

**Review by danbjoseph — APPROVED** [review]

**Pull request merged**